ads/google/imaVideo.js

```diff
@@ -378,12 +384,37 @@ export function imaVideo(global, data) {
           false);
     });
 
+    // Handle settings that need to be set before the AdDisplayContainer is
+    // created.
+    if (imaSettings.locale) {
```

Comment:
please use `imaSettings['locale']` format everywhere since tryParseJson returns a `JsonObject` and we like to enable a Closure Compiler optimization which will obfuscate properties